dynawo/sources/Modeler/DataInterface/DYNGeneratorInterface.h

```diff
@@ -150,8 +130,7 @@ class GeneratorInterface : public ComponentInterface {
   virtual void exportStateVariablesUnitComponent() = 0;
 
   /**
-   * @brief Retrieve the list of reactive curve points, if any
-   * @returns list of reactive curve points
+   * @copydoc ReactiveCurvePointsInterface::getReactiveCurvesPoints() const
```

Comment:
I would have called it getReactiveCurvePoints and not getReactiveCurvesPoints (no "s" at the end of Curve") to be consistent

Reply:
The previous API was named that way. To keep backwards compatibility, I kept the same name

Reply:
Ok, thanks